packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -48,6 +48,9 @@ const {
   measure: fabricMeasure,
   measureInWindow: fabricMeasureInWindow,
   measureLayout: fabricMeasureLayout,
+  unstable_DefaultEventPriority: FabricDefaultPriority,
+  unstable_DiscreteEventPriority: FabricDiscretePriority,
+  unstable_currentEventPriority: fabricGetCurrentEventPriority,
```

Comment:
I’m a bit confused, is this a field or a function?

Reply:
It is a function. It will return one of two values, either `unstable_DefaultEventPriority` or `unstable_DiscreteEventPriority`.

This is my first time doing anything in React repo. Is there a different syntax used for functions? I took inspiration from the functions above.

Reply:
I guess I'd expect it to be called `unstable_getCurrentEventPriority` to clarify it's a function.

Reply:
I'll rename it. Thanks for the suggestion!